front/pages/api/w/[wId]/assistant/agent_configurations/text_as_cron_rule.ts

```diff
@@ -50,11 +54,17 @@ async function handler(
       });
 
       if (r.isErr()) {
+        const cleanMessage = [
+          INVALID_TIMEZONE_MESSAGE,
+          TOO_FREQUENT_MESSAGE,
+        ].includes(r.error.message)
+          ? r.error.message
+          : GENERIC_ERROR_MESSAGE;
         return apiError(req, res, {
           status_code: 500,
           api_error: {
             type: "internal_server_error",
-            message: "Failed to generate cron rule.",
+            message: cleanMessage,
```

Comment:
I went the way of using a clean message in the API because i could not make the error code work.
If I use something which is not a internal_server_error the error object get transformed to their tostring at some point (probably a call to normalizeError)